{{cookiecutter.project_slug}}/.pre-commit-config.yaml

```diff
@@ -25,8 +34,8 @@ repos:
     rev: v0.3.0
     hooks:
     -   id: blacken-docs
-        additional_dependencies: [black]
-        types: [markdown, rst]
+        additional_dependencies: [black==18.9b0]
```

Comment:
If we hard-code that here we should make a note to ourselves in whatever ends up being our how-to-release document to update it.

Reply:
Blacken-docs does not run with newer black versions.